test/data/case5_restoration_iterative_shunt.m

```diff
@@ -0,0 +1,90 @@
+% used in tests of,
+% - restoration modeling without storage
```

Comment:
Is this comment correct?  It looks like storage is in this case.

I also suggest the the file name focus on what feature this data has compared to other test data, rather than a specific function it is used to test.

Reply:
comment changed to mention the added shunts in this case.